docs/topics/examples/simple-event/simple-event.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# The Kubernetes namespace in which Brigade is running.
+namespace="default"
+
+event_provider="simple-event"
+event_type="my_event"
+
+# This is github.com/deis/empty-testbed
+project_id="brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac"
+commit="master"
+
+
+# This is the brigade script to execute
+script=$(cat <<EOF
+const { events } = require("brigadier");
+events.on("my_event", (e) => {
+  console.log("The system time is " + e.payload);
+});
+EOF
+)
+
+# Now we will generate a new event evrey 60 seconds.
+while : ; do
+  # We'll use a UUID instead of a ULID. But if you want a ULID generator, you
+  # can grab one here: https://github.com/technosophos/ulid
+  uuid="$(uuidgen | tr '[:upper:]' '[:lower:]')"
```

Comment:
if you use `uuidgen -t` then your UUID is time based, thus sortable.

Reply:
My version of uuidgen does not have a `-t`. But you are right -- a UUIDv1 generator will make it at least sorta sortable.